content/preferences-settings/shortcuts.md

```diff
@@ -5,133 +5,204 @@ weight: 120
+
+You can alter the value of a slider more quickly or slowly than normal by defining the _speed_ of the action in the shortcut mapping screen. By default a _value_ (or _force_) effect is given a speed of 1.0, which means that it is changed at the default rate defined by the given slider. You can alter the slider more quickly by increasing the speed (a speed of 10 makes the action 10x faster) or more slowly by decreasing it (a speed of 0.1 makes the action 10x slower).
+
+# fallbacks
+
+Where a _widget_ can have multiple different _actions_ applied to it, it can be tedious to set up individual shortcuts for each one of those actions. To make this process simpler, if you create a simple shortcut a number of effects can be made available by default as extensions to that shortcut. These are known as _fallbacks_.
+
+While fallbacks are a powerful way to quickly set up multiple actions using predefined and consistent shortcuts, they will assign a lot of actions automatically (which might not be what you want), and can be hard to understand. Fallbacks are therefore disabled by default and you will need to click on the "enable fallbacks" check box in the shortcuts setup window to enable them.
+
+To take a brief example, you could create a simple shortcut (e.g. Ctrl+R) against a processing module. This will automatically set up the following _fallback_ effects using the defined shortcut, extended with mouse-clicks. In each case (except the first) you should hold the initial shortcut while clicking with your mouse. The final mouse-click will apply the action defined below:
 
-As with processing modules, some shortcut actions are provided by default for each module:
+- Ctrl+R (no mouse-click) to show/hide the module (the default fallback)
+- Ctrl+R+left-click to enable/disable the module
+- Ctrl+R+left-double-click to reset the module
+- Ctrl+R+right-click to show the module's preset menu
+- Ctrl+R+right-double-click to show the module's multiple instance menu
 
-show module
-: Expand or collapse the module.
+Similar fallbacks are defined for many common UI elements and all can be manually overridden.
 
-reset module parameters
-: Reset the module to its default state.
+Some fallback actions are defined using modifier keys (usually `Ctrl+` and `Shift+`). In this case you must define an initial shortcut without such a modifier in order to be able to use these fallbacks. For example, if you assign Ctrl+R to an action, you cannot use a `Ctrl+` fallback.
 
-show preset menu
-: Show the presets menu for the module.
+To see a list of the default fallbacks, click the "enable fallbacks" checkbox in the shortcut mapping screen, expand the "fallbacks" list in the top panel and select an option. An additional item (also named "fallbacks") will then appear in the bottom panel containing full details of the available fallbacks.
```

Comment:
To see _all_ fallbacks, you'd select the fallbacks _category_ in the top panel. To see fallbacks for any widget, for example a slider, just select that slider. You'll see all explicitly defined shortcuts for that individual slider and also fallbacks for the _slider_ type (and, in this specific case, for the _value_ type).